# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../faac/fix.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- faac-1.28-orig/common/mp4v2/mpeg4ip.h       2009-01-26 23:42:35.000000000 +0100
+++ facc-1.28/common/mp4v2/mpeg4ip.h    2009-11-28 18:30:58.720448072 +0100
@@ -123,7 +123,7 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-char *strcasestr(const char *haystack, const char *needle);
+//char *strcasestr(const char *haystack, const char *needle);
 #ifdef __cplusplus
 }
 #endif
